src/integration-test/tests/citizen/defence/steps/defence.ts

```diff
@@ -284,39 +284,21 @@ export class DefenceSteps {
         I.see('How did you pay the amount claimed?')
         break
 
+      case DefenceType.FULL_ADMISSION:
+        defendantSteps.selectTaskChooseAResponse()
+        defendantDefenceTypePage.admitAllOfMoneyClaim()
+        defendantSteps.selectTaskDecideHowWillYouPay()
+        defendantWhenWillYouPage.chooseImmediately()
+        break
+
       default:
         throw new Error('Unknown DefenceType')
     }
 
-    this.checkAndSendAndSubmit(defendantType)
-    if (defenceType === DefenceType.FULL_REJECTION_WITH_DISPUTE || defenceType === DefenceType.FULL_REJECTION_BECAUSE_FULL_AMOUNT_IS_PAID) {
+    if (defenceType === DefenceType.FULL_REJECTION_WITH_DISPUTE
```

Comment:
Why do we have this condition? Shouldn't all supported defence types end on check and send page?